src/javascript/binary/websocket_pages/user/ico_subscribe.js

```diff
@@ -109,6 +110,31 @@ const ICOSubscribe = (() => {
         $total.html(formatMoney(currency, total));
     };
 
+    const ICOcountDown = () => {
+        const timer = window.setInterval(() => {
+            const start_time = 1510704000; 
+            const current_time = Math.round((new Date()).getTime()/1000); 
```

Comment:
Need to check against GMT time as client time is not reliable. You can use `window.time` instead.